### PR TITLE
Drew.another - Update for 0.5.0 testnet

### PR DESCRIPTION
**Pull request by drewstone**
Adds tx weighting, fee calculations.

### CODE REVIEW

**Comment by drewstone** [comment]
@jnaviask if we want to integrate the testing file as well, we can or propagate it locally for testing.

File: node/runtime/src/lib.rs [hunk 1]
```diff
@@ -60,20 +61,31 @@ pub use runtime_primitives::{Permill, Perbill};
...
+
 // Make the WASM binary available.
 #[cfg(feature = "std")]
 include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
 
 /// This runtime version.
 pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("edgeware"),
-	impl_name: create_runtime_str!("edgeware-node"),
+	impl_name: create_runtime_str!("edgeware-Availab"),
```
Comment (jnaviask) [inline node/runtime/src/lib.rs]: what does this mean

File: node/service/src/chain_spec.rs [hunk 2]
```diff
@@ -183,18 +182,18 @@ pub fn testnet_genesis(
...
             phantom: Default::default(),
         }),
         elections: Some(ElectionsConfig {
-            members: vec![],
-            presentation_duration: 1 * DAYS,
-            term_duration: 28 * DAYS,
-            desired_seats: 0,
+            members: initial_authorities.iter().map(|x| (x.1.clone(), 1000000)).collect(),
+            desired_seats: 5,
+            presentation_duration: 25,
```
Comment (jnaviask) [inline node/service/src/chain_spec.rs]: These shouldn't be the final values for presentation/term duration.

File: node/runtime/src/lib.rs (hunk 1 above)
Comment (drewstone) [reply node/runtime/src/lib.rs]: Definitely goofed

File: node/service/src/chain_spec.rs (hunk 2 above)
Comment (drewstone) [reply node/service/src/chain_spec.rs]: What would you change them to?

**Pull request merged**